autoscale-kubernetes-container/src/main/config/cfg~caf~autoscaler~K8sAutoscaleConfiguration.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015-2021 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+({
+    maximumInstances: getenv("CAF_AUTOSCALER_MAXIMUM_INSTANCES") || 100,
+    namespaces: getenv("CAF_AUTOSCALER_KUBERNETES_NAMESPACES") || "default",
+    metric: getenv("CAF_AUTOSCALER_METRIC") || "rabbitmq",
```

Comment:
Can this be removed?  I don't understand RabbitMQ configuration in the K8s config file.

Reply:
It not rabbitmq cfg, its telling the autoscaler-source what value to look for in autoscale.metric

Reply:
I still don't get it, and I don't see any code calling `getMetric()` to use it.